### PR TITLE
Make query accept application/json on get transaction details

### PR DESCRIPTION
**Pull request by flasomm**
Hi,
I've juste a made a small change on get transaction details to accept json.
Do you mind to merge it on your branch master ?
Regards,
Fabrice Sommavilla

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 node_modules
 npm-debug.log
+.idea
```
Comment (ludoblues) [inline .gitignore]: Could you remove that?

**Review by ludoblues — CHANGES_REQUESTED** [review]
Ok for me, just need a small cleanup.

File: .gitignore (hunk 1 above)
Comment (flasomm) [reply .gitignore]: Well, if you want, but it's just a .gitignore file, nothing to worry about !
This is from my IDE.

Comment (flasomm) [reply .gitignore]: Done on the last commit, I've squashed it !

**Comment by flasomm** [comment]
Thank you ! Could you also please update the npm package https://www.npmjs.com/package/hipay ?

**Pull request merged**